pom.xml

```diff
@@ -183,6 +184,17 @@
             </dependency>
```

Comment:
This still contains the `arquillian-recorder-reporter-*` dependencies - technically, they shouldn't be necessary.